### DISCUSSION TITLE
Software to create the entity-relationship graph diagrams

### DISCUSSION THREAD

**Discussion by bodokaiser** [ent/ent#1468] [Q&A]
Hi,

first of all many thanks for ent! ent is a truly elegant solution to a hard problem in Go (before using ent, we were writing everything that ent now generates by hand)!

In the process of rewriting our backend with ent, I would like to create an entity-relationship graph diagram as used on the homepage.

What software was used to create the entity-relationship graph diagrams in [ent/assets](https://github.com/ent/assets/blob/master/entgo/er_following_followers.png)?

Best,
Bodo

**Comment by cliedeman** [comment]
Hi @bodokaiser I don't think its mentioned anywhere on the readme. it might have just been posted on slack but @a8m built this - https://github.com/a8m/enter

**Comment by rotemtam** [comment]
Hey 
I'm pretty sure the beautiful diagrams in the docs are handcrafted by @a8m using draw.io or similar. But you can get pretty great results with github.com/a8m/enter
